app/test_main.py

```diff
@@ -1,3 +1,29 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected_output",
+    [
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 28, [2, 2]),
+        (28, 29, [3, 3])
```

Comment:
The test case (27, 28, [2, 2]) seems to have an incorrect expected output. Since the dog age is one year more than the cat age, the expected output should reflect this difference. Please review this test case.